packages/block-library/src/post-navigation-link/block.json

```diff
@@ -57,6 +57,18 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			}
```

Comment:
```suggestion
```

The prevailing design feedback lately has been for nice applications of border support to make the border controls optional i.e. not displayed by default. 

The may indicator of that has been whether the spacing controls were displayed by default. This block doesn't even have spacing support yet. I'd take that as a sign this is very edge case territory and make the controls optional.

Reply:
We could include spacing support also in this PR, and will make border and spacing optional by default. I'll update the PR.